src/System.Memory/ref/System.Memory.cs

```diff
@@ -93,7 +93,6 @@ public static partial class MemoryExtensions
     {
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
-        public Memory(System.Buffers.MemoryManager<T> manager, int start, int length) { throw null; }
```

Comment:
These updates to the System.Memory ref need to be made to the System.Runtime ref as well (remove ctor and Length, add protected CreateMemory):
https://github.com/dotnet/corefx/blob/bd1911e9b3bf104fd9821a4fc68ea1baafe780ef/src/System.Runtime/ref/System.Runtime.cs#L1693
https://github.com/dotnet/corefx/blob/bd1911e9b3bf104fd9821a4fc68ea1baafe780ef/src/System.Runtime/ref/System.Runtime.cs#L3738